src/tile/tile.scss

```diff
@@ -153,7 +153,7 @@
 
 @mixin iui-tile-thumbnail {
   width: 100%;
-  height: $iui-baseline * 14;
+  height: $iui-baseline * 14 + 1px; // `+ 1 px` to account for border width
```

Comment:
Same.. return `content-box` or keep the size as previous? But here ight be sensitive, cause border is only on the bottom..